src/awkward/_connect/_autograd.py

```diff
@@ -15,7 +15,8 @@
 def register():
     import autograd
 
-    global NEP13Box
+    # PyLint false positive: https://github.com/PyCQA/pylint/issues/5756
+    global NEP13Box  # pylint: disable=global-variable-not-assigned
```

Comment:
Will be removable, see https://github.com/PyCQA/pylint/issues/5756

Reply:
This autograd module will not survive into version 2.0. JAX can be seen as an update of autograd, and @swishdiff will be integrating Awkward with that instead. (I originally went for autograd because its pure-Pythonness made it an easier target.)